services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/utils/GoalStateManager.java

```diff
@@ -46,418 +46,418 @@
 public class GoalStateManager {
     public static final int FORMAT_REVISION_NUMBER = 1;
     @Autowired private GoalStateService goalStateService;
-  private static final Logger LOG = LoggerFactory.getLogger();
-
-  private void printNetworkConfiguration(NetworkConfiguration networkConfiguration)
-  {
-    LOG.log(Level.INFO,
-            "### networkConf str: "+networkConfiguration.toString());
-    ExclusionStrategy myExclusionStrategy =
-            new ExclusionStrategy() {
-              @Override
-              public boolean shouldSkipField(FieldAttributes fa) {
-                return fa.getName().equals("tenantId");
-              }
-
-              @Override
-              public boolean shouldSkipClass(Class<?> clazz) {
-                return false;
-              }
-            };
-    Gson gson = new GsonBuilder().setExclusionStrategies(myExclusionStrategy).create();
-    LOG.log(Level.INFO,"###############");
-    LOG.log(Level.INFO,gson.toJson(networkConfiguration));
-  }
-  /**
-   * transform client of dpm msg to aca protobuf format
-   *
-   * @param networkConfiguration  msg to be transformmed
-   * @return Map<String, Goalstate.GoalState>
-   * @throws RuntimeException Various exceptions that may occur during the send process
-   */
-  public Map<String, Goalstate.GoalState> transformNorthToSouth(
-      NetworkConfiguration networkConfiguration) throws RuntimeException {
-    // print entry input
-    printNetworkConfiguration(networkConfiguration);
-
-    Map<String, Set<String>> portsInSameSubnetMap = new HashMap<>();
-
-    Map<String, Set<NeighborInfo>> neighborInfoInSameSubenetMap = new HashMap<>();
-
-    InternalPortEntity[] portStatesArr =
-        networkConfiguration.getPortEntities().toArray(new InternalPortEntity[0]);
-    InternalSubnetEntity[] subnetArr =
-        networkConfiguration.getSubnets().toArray(new InternalSubnetEntity[0]);
-    com.futurewei.alcor.web.entity.vpc.VpcEntity[] vpcArr =
-        networkConfiguration.getVpcs().toArray(new com.futurewei.alcor.web.entity.vpc.VpcEntity[0]);
-
-    // TODO need to v2 subnet and vpc part when logic is
-    //  clear and integration done
-    Map<String, List<InternalPortEntity>> mapGroupedByHostIp = new HashMap();
-    Map<String, InternalSubnetEntity> subnetMap = new HashMap<>();
-    Map<String, InternalPortEntity> portMap = new HashMap<>();
-    Map<String, com.futurewei.alcor.web.entity.vpc.VpcEntity> vpcMap = new HashMap<>();
-    // construct map from list
-    for (InternalSubnetEntity s : subnetArr) {
-      subnetMap.put(s.getId(), s);
+    private static final Logger LOG = LoggerFactory.getLogger();
+
+    private void printNetworkConfiguration(NetworkConfiguration networkConfiguration)
```

Comment:
I think we could revert the change as it is related to PR #418 .

Reply:
Sure

Reply:
> I think we could revert the change as it is related to PR #418 .

@kevin-zhonghao I think this PR still contains payload from PR #418, which should not be included in this PR.